src/snkit/network.py

```diff
@@ -28,6 +29,9 @@
 
 from collections import Counter
 
+# configure logging with a timestamp and process id prefix
+logging.basicConfig(format="%(asctime)s %(process)s %(message)s", level=logging.INFO)
```

Comment:
Suggest removing this line - I think logging configuration should be the caller's (library user's) concern, not up to us to set here.

Reply:
Ok, done